starboard/android/shared/BUILD.gn

```diff
@@ -322,6 +322,7 @@ static_library("starboard_platform") {
   ]
 
   deps = [
+    "//base/third_party/symbolize:symbolize",
```

Comment:
just a reminder that `//base/third_party/symbolize:symbolize` is the same as `//base/third_party/symbolize`